### PR TITLE
Renamed visibility presets to map themes

### PR DESCRIPTION
**Pull request by anitagraser**
This is the first of multiple files that need to be updated to address #1229

### CODE REVIEW

**Comment by anitagraser** [comment]
One question about consistency: Show it be "map theme" or "Map Theme"?

**Comment by yjacolin** [comment]
@anitagraser could you make the small change from the @DelazJ review? :)

File: source/docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -60,8 +60,8 @@ At the top of the Layers panel, a toolbar allows you to:
 * |symbology| :sup:`Open the layer styling dock`: toggle the layer styling
   panel on and off.
 * |addGroup| :sup:`Add new group`
-* |showPresets| :sup:`Manage Visibility`: control visibility of layers and
-  preset layers combination.
+* |showPresets| :sup:`Manage Map Themes`: control visibility of layers and
```
[the remark this answers is not in the document]
Comment (anitagraser) [reply source/docs/user_manual/introduction/general_tools.rst]: It would probably be better for the future to rename it

**Comment by anitagraser** [comment]
@yjacolin Thank you for the reminder. I've fixed everything except for the substitution. Waiting for your recommendation.

**Comment by DelazJ** [comment]
I added the substitution. Thanks @anitagraser

**Pull request merged**